types/core/speedy-media-source.d.ts

```diff
@@ -73,5 +73,5 @@ export class SpeedyMediaSource {
      */
     _load(element: SpeedyMediaSourceNativeElement): SpeedyPromise<SpeedyMediaSource>;
 }
-export type SpeedyMediaSourceNativeElement = HTMLImageElement | HTMLVideoElement | HTMLCanvasElement | ImageBitmap;
+export type SpeedyMediaSourceNativeElement = HTMLImageElement | HTMLVideoElement | HTMLCanvasElement | OffscreenCanvas | ImageBitmap;
```

Comment:
i think i should add also ImageData to SpeedyMediaSourceNativeElement. I noticed that tsc added some changes, maybe tpes definitions wasn't updated?